models/run_results.yml

```diff
@@ -1,6 +1,85 @@
 version: 2
 
 models:
+  - name: dbt_invocations
+    description: >
+      Attributes associated with each dbt invocation. Inserted at the end of each invocation.
+
+    columns:
+      - name: invocation_id
+        data_type: string
+        tests:
+          - not_null
+          - unique
+          - relationships:
+              field: invocation_id
+              to: ref('dbt_run_results')
+
+      - name: run_started_at
+        data_type: timestamp
+        description: Timestamp the invocation was started.
+
+      - name: run_completed_at
+        data_type: timestamp
+        description: Timestamp the invocation was completed
+
+      - name: generated_at
+        data_type: timestamp
+        description: Timestamp the invocation was completed and generated.
+
+      - name: command
+        data_type: string
+        description: dbt command that was used. For example, run.
+
+      - name: dbt_version
+        data_type: string
+        description: Version of dbt that was used in this invocation.
+
+      - name: elementary_version
+        data_type:
+        description:
+
+      - name: full_refresh
+        data_type: boolean
+        description: Whether or not this invocation was executed with a full-refresh flag
+
+      - name: cli_vars
+        data_type: string
+        description: Dictionary of the variables (and values) that were declared in the invocation.
+
+      - name: vars
+        data_type: string
+        description: Dictionary of all variables (and values) in the dbt project. If none were declared at runtime, these are the variables declared in dbt_project yml
+
+      - name: target_name
+        data_type: string
+        description: Name of the target used in this invocation.
+
+      - name: target_database
+        data_type: string
+        description: Name of the target schema that was used in this invocation.
+
+      - name: target_schema
+        data_type: string
+        description: Name of the target schema that was used in this invocation.
+
+      - name: target_profile_name
+        data_type: string
+        description: Name of the dbt profile that was used in this invocation.
+
+      - name: threads
+        data_type: integer
+        description: ""
+
+      - name: selected
+        data_type: string
+        description: While this is a string in the database, this can easily be converted to an array.
```

Comment:
I'd add to the description - 'the selected resources in the dbt command. While this is a string in the database, this can easily be converted to an array.'